pkg/cri/server/sandbox_stats_list.go

```diff
@@ -34,6 +36,10 @@ func (c *criService) ListPodSandboxStats(
 	podSandboxStats := new(runtime.ListPodSandboxStatsResponse)
 	for _, sandbox := range sandboxes {
 		sandboxStats, err := c.podSandboxStats(ctx, sandbox)
+		if errdefs.IsUnavailable(err) {
+			log.G(ctx).WithField("podsandboxid", sandbox.ID).Warnf("failed to get pod sandbox stats, this maybe a transient error: %v", err)
```

Comment:
suggest debug here vs warning.. transient isn't really an error or warning just opportunistic timing... a warning could be noisy for admins...

Reply:
Updated